Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Android/Setup.cs

```diff
@@ -19,19 +27,29 @@ protected override IMvxIoCProvider CreateIocProvider()
         {
             var containerBuilder = new ContainerBuilder();
             containerBuilder.RegisterModule<ApplicationModule>();
+            containerBuilder.Register(c => Environment.GetFolderPath(Environment.SpecialFolder.Personal)).Named<string>(App.BootstrapParamaters.LocalAppDataFolder);
```

Comment:
> That const is defined at application level (platform-independent), as:

Ahhhhhhhhhh got it. Sorry, it looked so familiar that I thought we were actually reading the value from the environment. My bad.